packages/admin/admin-theme/src/componentsTheme/MuiTypography.ts

```diff
@@ -3,6 +3,14 @@ import { GetMuiComponentTheme } from "./getComponentsTheme";
 
 export const getMuiTypography: GetMuiComponentTheme<"MuiTypography"> = (component) => ({
     ...component,
+    defaultProps: {
+        ...component?.defaultProps,
+        variantMapping: {
+            ...component?.defaultProps?.variantMapping,
+            list: "ul",
+            listItem: "li",
```

Comment:
I think the values passed in from the application-theme should be added to the end so they can override our values. 

```suggestion
            list: "ul",
            listItem: "li",
            ...component?.defaultProps?.variantMapping,
```